js/webview/swipeEvents.js

```diff
@@ -8,6 +8,9 @@ var beginningScrollRight = null
 
 var hasShownSwipeArrow  = false
 
+// TODO: find out a way to read this from settings
+var isZoomWithScrollEnabled = true
```

Comment:
@PalmerAL, Could you please help me figure out how to read the respective setting's value in this file?

Reply:
@revolter You can't, because you can only read from the database in a page with a file:/// origin. The way we worked around this for the swipe preference was to send an IPC to the parent process, which would then check the preference and do the navigation. (the code for this is here: https://github.com/minbrowser/min/blob/master/js/webviews.js#L272-L278). I'm guessing you'd have to do something similar here.